scripts/hello-world-deploy.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/sh
 
+pacman -Sy
+
+if ( pacman -Qu )
```

Comment:
You want to practice doing "desired state" which means putting the conditional the "other way" which is "if there are no packages to be upgraded, then I am at the latest. Think of it again as there should be no actions in the "then" part of any conditional, but only take action in the else, because the desired state has not been achieved.

Might want something more like `test -z $(pacman -Qu)` which is "there should be nothing to upgrade".

Reply:
tried `test -z $(pacman -Qu)` but when there is something to upgrade test gives the error "too many arguments" which from reading a couple stack threads is because it reads white space or new lines between packages as a separate argument. There are ways to take that error (which returns a 2) but it looks clunky to me. 


If you don't like the ! (not) can you tell me why. I believe it falls within the 'desired state' we're shooting for.